db/column_family.cc

```diff
@@ -1118,22 +1116,19 @@ SuperVersion* ColumnFamilyData::GetThreadLocalSuperVersion(
   if (sv == SuperVersion::kSVObsolete ||
       sv->version_number != super_version_number_.load()) {
     RecordTick(ioptions_.statistics, NUMBER_SUPERVERSION_ACQUIRES);
-    SuperVersion* sv_to_delete = nullptr;
 
     if (sv && sv->Unref()) {
       RecordTick(ioptions_.statistics, NUMBER_SUPERVERSION_CLEANUPS);
-      db_mutex->Lock();
+      db->mutex()->Lock();
       // NOTE: underlying resources held by superversion (sst files) might
       // not be released until the next background job.
-      sv->Cleanup();
```

Comment:
It feels more natural to keep this line here than moving it to DB::AddSuperVersionsToFreeQueue().